src/utils/trust/who.Is.Involved.mapper.ts

```diff
@@ -0,0 +1,18 @@
+import * as Trust from 'model/trust.model';
```

Comment:
file name needs to be in lower case
 Is.Involved - > is.involved

Reply:
@markpit - File renamed